Edit README.md

teru0225 opened the pull request:
Edit README.md
- you can see the construction of the tables for Chat-space

teru0225 committed:
Edit .gitignore

edited .gitignore

teru0225 committed:
Edit README.md

Edit README.md
- you can see the construction of the tables for Chat-space

teru0225 committed:
Edit README

There were 2 mistakes
- Unnecessary dot
- wrong space

ReoKuriyama commented on README.md:
```diff
@@ -1,24 +1,53 @@
+|name|string|null: false|
+|email|varchar|null: false|
+|group_id|integer||
+|password|string|null: false|
```
deviseにすでに含まれているカラムは今回除きましょう

ReoKuriyama commented on README.md:
```diff
@@ -1,24 +1,53 @@
+- has_many :group_users
+- has_many :messages
+- has_many :users, through: :group_users
+- accepts_nested_attributes_for :group_users
```
こちらdbの設計には書かなくて大丈夫です。

teru0225 committed:
Edit README

deleted
- password column
(Unnecessary to write down columns that devise gem makes)
- accepted_nested_attributes_for
(Unnecessary to write down on README)

ReoKuriyama commented on README.md:
```diff
@@ -1,24 +1,50 @@
+|------|----|-------|
+|name|string|null: false|
+|email|varchar|null: false|
+|group_id|integer||
```
見逃していましたが、ここにgroup_idは必要ないですね

ReoKuriyama commented:
コメントしたところを修正してもらったらLGTMです

teru0225 commented:
> コメントしたところを修正してもらったらLGTMです

かしこまりました
ありがとうございます
